src/polyglot-notebooks-vscode-common/tests/metadataUtilities.test.ts

```diff
@@ -703,6 +702,227 @@ describe('metadata utility tests', async () => {
+                                    "aliases": [
+                                        "powershell"
+                                    ],
+                                    "languageName": "PowerShell",
+                                    "name": "pwsh"
+                                },
+                                {
+                                    "aliases": [
+                                    ],
+                                    "languageName": null,
+                                    "name": "value"
+                                },
+                                {
+                                    "aliases": [
+                                        "frontend"
+                                    ],
+                                    "languageName": null,
+                                    "name": "vscode"
+                                },
+                                {
+                                    "aliases": [
+                                    ],
+                                    "name": "webview"
+                                }
+                            ]
+                        }
+                    }
+                },
+                "cells": [
+                ],
+                "nbformat": 4,
+                "nbformat_minor": 2
+            },
+            "indentAmount": " "
+        };
+
+        metadataUtilities.sortAndMerge(destination, source);
+
+        expect(destination).to.deep.equal({
+            custom:
+            {
+                cells: [],
+                metadata:
+                {
+                    kernelspec:
+                    {
+                        display_name: '.NET (C#)',
+                        language: 'C#',
+                        name: '.net-csharp'
+                    },
+                    polyglot_notebook:
+                    {
+                        kernelInfo:
+                        {
+                            defaultKernelName: 'powershell',
+                            items:
+                                [{ aliases: [], languageName: null, name: '.NET' },
+                                { aliases: ['C#', 'c#'], languageName: 'C#', name: 'csharp' },
+                                { aliases: ['F#', 'f#'], languageName: 'F#', name: 'fsharp' },
+                                { aliases: [], languageName: 'HTML', name: 'html' },
+                                { aliases: ['js'], languageName: 'JavaScript', name: 'javascript' },
+                                { aliases: [], languageName: 'Mermaid', name: 'mermaid' },
+                                { aliases: [], name: 'powershell' },
+                                { aliases: ['powershell'], languageName: 'PowerShell', name: 'pwsh' },
+                                { aliases: [], languageName: null, name: 'value' },
+                                { aliases: ['frontend'], languageName: null, name: 'vscode' },
```

Comment:
We correct for `languageName: null` now, so this test should change.  E.g., `languageName: null` shouldn't appear anywhere any more.